### DISCUSSION TITLE
8月6日续费了2年的专业版，至8月9日是正常的，8月12日早上显示为免费版，授权专业版，但是邮箱无法收到反馈，无法获得验证码。

### DISCUSSION THREAD

**Discussion by johnny6143** [getlantern/lantern-forum-cn#880] [Q&A]
8月6日续费了2年的专业版，至8月9日是正常的，8月12日早上显示为免费版，授权专业版，但是邮箱无法收到反馈，无法获得验证码。
8月12日下午1点19分发送 邮件至  mike@getlantern.org  内有付款凭证。
注册蓝灯的邮箱用的是qq邮箱，如大陆限制，    请帮忙反馈验证码 至  ohnny130419@gmail.com 邮箱。
![Uploading ccea9054de4e6a2c3e9670e8c1e3f3c.jpg…]()

**Comment by johnny6143** [comment]
![ccea9054de4e6a2c3e9670e8c1e3f3c](https://github.com/user-attachments/assets/3fefefac-e638-4a3f-8b49-9b17c8fc346d)

**Comment by johnny6143** [comment]
有消息了吗？

**Comment by allenwork0001** [comment]
现在soft003.com不能买了吗？优惠没了？

**Reply by johnny6143** [reply]
？这个是啥？

**Comment by MaxMKMax** [comment]
@johnny6143 您好. 请您等一下. 这几天支付部门的人不在线. 我帮您提交问题上去. 谢谢您的耐心等待. 
已经帮您提交好了!

**Reply by johnny6143** [reply]
ok，那最新情况怎样了呢？

**Reply by greySkiesMarsh** [reply]
Hi @johnny6143 

Due to what seems to be a database error, I wasn't able to locate your account in our system. Please redeem the following code with the email address you've posted in this thread, and I can set the expiration to reflect your purchase. 

JBDMH-G3QDK-XVWMY-JV3WT-2RXVH

**Reply by MaxMKMax** [reply]
@johnny6143 由于似乎是数据库错误，我无法在我们的系统中找到您的帐户。请使用您在此主题中发布的电子邮件地址兑换以下代码，我可以设置有效期以反映您的购买。

JBDMH-G3QDK-XVWMY-JV3WT-2RXVH

**Reply by johnny6143** [reply]
收到，谢谢，已经使用兑换码恢复了专业版，系统中的账户应该已经恢复了吧？专业版到期日期为2024年10月26日，还剩1个月啦，麻烦把有效期延长2年~~谢谢。

**Reply by johnny6143** [reply]
@Mike777Khoo  收到，谢谢，已经使用兑换码恢复了专业版，系统中的账户应该已经恢复了吧？目前我的蓝灯专业版到2024年10月26日到期，还剩1个月啦，  麻烦参照我的付款凭证 把有效期延长2年~~谢谢。

**Comment by johnny6143** [comment]
@Mike777Khoo 已经恢复了专业版，目前我的蓝灯专业版到2024年10月26日到期，还剩2个周了， 麻烦参照2024-8-6日的付款凭证 把有效期延长2年~~谢谢。

**Comment by johnny6143** [comment]
@Mike777Khoo 已经恢复了专业版，目前我的蓝灯专业版到2024年10月26日到期，还剩2个周了， 麻烦参照2024-8-6日的付款凭证 把有效期延长2年~~谢谢。

**Comment by johnny6143** [comment]
@Mike777Khoo 已经恢复了专业版，目前我的蓝灯专业版到2024年10月26日到期，还剩10天了， 麻烦看2024-8-6日的付款凭证 把有效期延长2年~~谢谢。

**Comment by johnny6143** [comment]
@Mike777Khoo 已经恢复了专业版，目前我的蓝灯专业版到2024年10月26日到期，还剩8天了， 麻烦看2024-8-6日的付款凭证 把有效期延长2年~~谢谢。

**Comment by johnny6143** [comment]
@Mike777Khoo 已经恢复了专业版，目前我的蓝灯专业版到2024年10月26日到期，还剩7天了， 麻烦看2024-8-6日的付款凭证 把有效期延长2年~~谢谢。

**Reply by MaxMKMax** [reply]
@johnny6143 您好. 好的, 我会帮您提交给负责人帮您处理. 但要等到下周二了, 负责人病假. 不在线. 谢谢您的耐心等待.

**Reply by johnny6143** [reply]
已经周四啦~，负责人情况怎样啊在线了吗？明天就到期了呢~~

**Reply by johnny6143** [reply]
@Mike777Khoo  今天登陆已经显示过期了，账号75769747@qq.com

**Comment by johnny6143** [comment]
今天已经过期了，麻烦恢复有效期， 账号 ***@***.***




蟹壳
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年10月19日(星期六) 凌晨0:51
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 8月6日续费了2年的专业版，至8月9日是正常的，8月12日早上显示为免费版，授权专业版，但是邮箱无法收到反馈，无法获得验证码。 (Discussion #880)





 
@johnny6143 您好. 好的, 我会帮您提交给负责人帮您处理. 但要等到下周二了, 负责人病假. 不在线. 谢谢您的耐心等待.

**Comment by johnny6143** [comment]
@Mike777Khoo    已经显示过期了，账号[75769747@qq.com](mailto:75769747@qq.com)

**Comment by johnny6143** [comment]
@Mike777Khoo 已经显示过期了，账号75769747@qq.com

**Comment by greySkiesMarsh** [comment]
Hello @johnny6143 

I sincerely apologize for the delay in resolving this matter. Your account has been restored with 3 complimentary months of Pro access added on.

**Reply by johnny6143** [reply]
谢谢，已经恢复正常了~~感谢~

**Comment by MaxMKMax** [comment]
@johnny6143 非常荣幸有像您这样通情达理的用户. 可以帮到您解决问题我们也很开心! 谢谢您支持蓝灯!

**Reply by johnny6143** [reply]
已经用了很多年蓝灯大概有6、7年了，收费的只要服务跟上，对你们还是很认可的。
